index.html

```diff
@@ -9,9 +9,10 @@
     <link href="css/normalize.css" rel="stylesheet">
     <link href="css/dark.css" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
-
```

Comment:
Do you mind if we preserve whitespace in places like this? Helps me ADHD-wise if I can see visual markers separating different things.

I'll work round it if that messes with you